flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-paimon/src/main/java/org/apache/flink/cdc/connectors/paimon/sink/SchemaChangeProvider.java

```diff
@@ -65,7 +65,7 @@ public static List<SchemaChange> add(AddColumnEvent.ColumnWithPosition columnWit
                                             "%s.%s.%s",
                                             CoreOptions.FIELDS_PREFIX,
                                             column.getName(),
-                                            CoreOptions.DEFAULT_VALUE_SUFFIX);
+                                            SchemaChangeProvider.DEFAULT_VALUE_SUFFIX);
                             result.add(SchemaChangeProvider.setOption(key, value));
```

Comment:
We should update the way of setting default value using newly added API.
```
        Optional.ofNullable(column.getDefaultValueExpression())
                .ifPresent(
                        value -> {
                            result.add(
                                    SchemaChange.updateColumnDefaultValue(new String[]{column.getName()}, column.getDefaultValueExpression()));
                        });
```

Reply:
No additional dependencies required, SchemaChange was already used in this class.